app/styles/site-layouts.scss

```diff
@@ -1,3 +1,22 @@
-body {
- padding: 10px;
+$app-qgreen: rgba(98,144,128,0.25);
+$app-halfgreen: rgba(98,144,128,0.5);
```

Comment:
constants need to be saved in the constants style sheet file i believe it's called constants.scss or you can do it in  a helpers.scss files. once they're there you can import them in nested sheets.